Source/WebCore/editing/TextIterator.cpp

```diff
@@ -1645,6 +1667,28 @@ String foldQuoteMarks(const String& stringToFold)
     result = makeStringByReplacingAll(result, leftSingleQuotationMark, '\'');
     result = makeStringByReplacingAll(result, leftLowSingleQuotationMark, '\'');
     result = makeStringByReplacingAll(result, rightDoubleQuotationMark, '"');
+    result = makeStringByReplacingAll(result, singleLow9QuotationMark, '\'');
```

Comment:
Now that we have so many additional quote types, I don’t think we should use a separate function call for each. If you like I can help write a version that is more efficient.